lib/tutorials/en_US/cookies.md

```diff
@@ -73,13 +112,14 @@ The `request.state` object contains the parsed HTTP state. Each key represents t
 const value = request.state.data;
 // console.log(value) will give you { firstVisit : false }
```

Comment:
This may not be true anymore when overriding the cookie value to `test`. Should we highlight the assumption that the initial cookie value wasn’t updated?